EUDIW trust infrastructure (#1192)
italia/eidas-it-wallet-docs: docs/en/trust-artifact-eudiw.rst

The Wallet-Relying Party Registration Certificate MUST be signed with the private key of Provider of the Wallet-Relying Party Registration Certificates.
In particular:

- The JWT MUST be signed with a JSON Advanced Electronic Signature with the B-B profile as defined in `ETSI TS 119 182-1`_.
- The CWT MUST be signed with an Advanced Electronic Signature following structure as defined in :rfc:`9052` and :rfc:`9360`.

Below a non-normative example of WRPRC header and payload for a Relying Party.

.. literalinclude:: ../../examples/wrprc-jwt-header.json
  :language: json

.. literalinclude:: ../../examples/wrprc-payload-ci.json
  :language: json

Below a non-normative example of WRPRC payload for a Relying Party Intermediary.

.. literalinclude:: ../../examples/wrprc-payload-rpi.json
  :language: json

.. warning::

  `ETSI TS 119 475`_, Table 10 defines the intermediary name subfield as ``sname``.
  The example in Annex C of the same standard uses ``name`` instead.
  This specification follows the normative Table 10 and uses ``sname``.

Trusted List, Lists of Trusted Lists, and Lists of Trusted Entities
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> This section describes the format and contents of three types of Trust Artifacts, each of which conveys a list of current and historical Trust Anchors (containers of an Entity public key and identifier which are assumed to be trusted).

Reviewer: ```suggestion
This section describes the format and contents of three types of Trust Artifacts, each of which conveys a list of current and historical Trust Anchors (containers of cryptographic materials and identifiers belonging to trusted Entities).
```

Ecosystem Entities utilize these lists to:

- **Validate runtime trustworthiness**: Verify a Trust Anchor (see :ref:`infrastructure-trust:Trust Anchor Certificate Profile`) to authenticate, authorize, or validate an entity or artifact during live operations.
- **Perform historical validation**: Validate information contained within the list for historical audit purposes.

The three distinct types of trust lists are:

> - Trusted Lists (TLs): Established under Chapter II of Annex I of CID (EU) 2015/1505, as amended by CID (EU) 2025/2164, and specified in `ETSI TS 119 612`_.

Reviewer: Annex I of CID (EU) 2015/1505, as amended by CID (EU) 2025/2164,

should be provided with their HREF as well

  Each Member State publishes one TL in XML format.
  It is signed by the respective Member State with an XAdES digital signature at conformance level baseline B (as defined in `ETSI EN 319 132-1`_).
  TLs are published in a machine-readable format at endpoints specified within the LOTL.
  These Lists hold current and historical information about the accreditation of trust service providers, referencing:

  - Qualified Trust Service Providers (QTSP)s, such as Qualified Certificates Issuing and revocation mechanisms, QEAA Providers, Qualified electronic archiving services.
  - Non-Qualified Trust Services such as EAA Providers.
  - Other Trust Services defined at the national level, such as archiving.

   Within eIDAS, TLs are maintained by Member States, who are responsible for keeping record of the trusted services providers under their respective jurisdiction.
   They are numbered and renewed periodically, and published in a website for unrestricted download.
   To protect their integrity and assure authenticity, they are also signed with trusted certificates contained in the LOTL.

> - List of Trusted Lists (LOTL): Established under Chapter II of Annex I of CID (EU) 2015/1505, as amended by CID (EU) 2025/2164, and specified in `ETSI TS 119 612`_.

Reviewer: Chapter II of Annex I of CID (EU) 2015/1505, as amended by CID (EU) 2025/2164

they should be referenced with their HREFs

  There is only one LOTL, which is published in XML format and signed by the European Commission (EC).
  It utilizes an XAdES digital signature at conformance level baseline B (per `ETSI EN 319 132-1`_) and references the trusted certificates that each National Trusted List.
  To facilitate key rotation and continuous updates, the LOTL implements a pivoting mechanism.

>   It is published in a machine-readable format at an endpoint specified within the Official Journal of the European Union (OJEU).

Reviewer: Official Journal of the European Union (OJEU) should be referenced with its HREF

  The XML schema for both Trusted Lists and List of Trusted Lists, containing parameters' name and description can be found at https://forge.etsi.org/rep/esi/x19_612_trusted_lists/-/raw/v2.4.1/19612_xsd.xsd. Currently, the human-readable version of the LOTL and National TLs are published in the following URI: <https://ec.europa.eu/tools/lotl/eu-lotl.xml>.

> - Lists of Trusted Entities (LoTE): Established under Articles 4 and 5 of [CIR 2024/2980] and specified in `ETSI TS 119 602`_.

Reviewer: CIR 2024/2980 should be referenced with its HREF

  These are available in either XML or JSON format and are signed with an AdES digital signature at conformance level baseline B (per `ETSI TS 119 182-1`_).

>   To facilitate continuous updates, the LoTE implements a pivoting mechanism and is published in a machine-readable format at an endpoint specified within the OJEU.

Reviewer: OJEU should be defined in the acronyms

  The LoTE types can be one of the following, as defined in annex C.2:

  - PID Provider;
  - Wallet Provider;
  - Provider of Wallet Relying Party Access Certificates;
  - Providers of Wallet Relying Party Registration Certificates;
  - Public sector bodies issuing Electronic Attestations of Attributes;
  - List of Registrars and Registers.

>   The following repository provides the normative JSON and XML schemas required for implementing the List of Trusted Entities https://forge.etsi.org/rep/esi/x19_60201_lists_of_trusted_entities.

Reviewer: ```suggestion
  The following repository provides the normative JSON and XML schemas required for implementing the List of Trusted Entities ``https://forge.etsi.org/rep/esi/x19_60201_lists_of_trusted_entities``.
```
